frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.html

```diff
@@ -1,8 +1,14 @@
+
+<!-- <form [formGroup]="applicationRulesForm" class="usa-form-large" novalidate> -->
```

Comment:
Can we clean up / remove any commented code

Reply:
Commented code removed